subprojects/core/src/main/java/org/gradle/api/internal/tasks/properties/ValidationActions.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 the original author or authors.
+ * Copyright 2019 the original author or authors.
```

Comment:
Nit: many such changes.

Reply:
I think I fixed them all. Note that some of them are shown up in the diff since git seems to detect some files as moved which don't have anything to do with each other.